php/libraries/Project.class.inc

```diff
@@ -341,10 +329,28 @@ class Project
      */
     public function updateRecruitmentTarget(?int $target = null): void
     {
-        $this->_updateField(
-            "recruitmentTarget",
-            $target ?? $this->_recruitmentTarget
-        );
+        if (isset($target)) {
+            $this->setRecruitmentTarget($target);
+        }
```

Comment:
I think this value will always be set, either to the value passed or `null`.

Reply:
this should be doing the same thing as ur null coalesce, i dont think the behaviour changes.

it defaults to the classes' _recruitmentTarget if null is passed and if the classes in null it sets it to null i guess

Reply:
Null coalesce handles `null` and not set. `isset()` only handles not set.

Reply:
When PHP 7.4 comes out we can use typed class properties so this won't be an issue. 😄

Reply:
https://www.php.net/manual/en/function.isset.php

isset() only returns false when a value is NULL

Reply:
My bad. Don't know why I thought that wasn't the case but I definitely have for a long time.

Thanks

Reply:
because PHP doesnt make sens

$x=null
isset($x) -> flase

:confused: